benchmark/local-udp-throughput

```diff
@@ -16,13 +55,28 @@ def send_metrics(client)
   client.service_check("StatsD.service_check", "ok")
 end
 
+def send_metrics_high_cardinality(client)
```

Comment:
What would be interesting would be to just have an argument or env var to define the cardinality / duplication count.

Reply:
This what `SERIES_COUNT` is controlling, how many different series we are sending. Is that what you meant?

For example, for SERIES_COUNT=2

we are going to send a counter with `#series:1` and another with `series:0` for each `ITERATIONS`.

Reply:
Yeah, I don't care so much how it's done, All I'm saying is that we should be careful to make comparisons with a fixed number of emitted metrics. What we should compare is performance on a given duplicate ratio.